scriptmodules/supplementary/moonlight-embedded.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+# @autor: Nyaran
+# @date: 2016/10/09
+# @usage: Place this script into RetroPie-Setup/scriptmodules/supplementary directory and execute the following command:
+#         `sudo RetroPie-Setup/retropie_packages.sh moonlight-embedded`
+#         Also you can use the setup script. the installer is located under *exp packages* option.
+
+
+rp_module_id="moonlight-embedded"
+rp_module_desc="Moonlight Embedded Game Streaming"
+rp_module_section="exp"
+
+function depends_moonlight-embedded() {
+    getDepends libopus0 libexpat1 libasound2 libudev0 libavahi-client3 libcurl3 libevdev2 libenet7 libraspberrypi0 libssl-dev libopus-dev libasound2-dev libudev-dev libavahi-client-dev libcurl4-openssl-dev libevdev-dev libexpat1-dev libpulse-dev uuid-dev libenet-dev
+}
+
+function sources_moonlight-embedded() {
+    gitPullOrClone "$md_build/installer" "https://github.com/Nyaran/RetropieMoonlightInstaller.git"
+    gitPullOrClone "$md_build/moonlight" "https://github.com/irtimmer/moonlight-embedded.git"
+}
+
+# Compile CMake 3.1+ until is not available on Raspbian repositories
+function build_cmake() {
+    wget https://cmake.org/files/v3.6/cmake-3.6.2.tar.gz
+    tar -xvf cmake-3.6.2.tar.gz
+    rm cmake-3.6.2.tar.gz
+    pushd cmake-3.6.2/
+    ./bootstrap
+    make
+    popd
+}
+
+function build_moonlight-embedded() {
+    build_cmake
+
+    pushd moonlight
+    git submodule update --init
+    mkdir build
+    cd build/
+
+    ../../cmake-3.6.2/bin/cmake ../
+    make
+    popd
+}
+
+function install_moonlight-embedded() {
+    # Install moonlight-embedded simple theme
+    mkdir -p /etc/emulationstation/themes/simple/moonlight-embedded/art
+    cp -v "$md_build/installer/themes/simple/theme.xml" /etc/emulationstation/themes/simple/moonlight-embedded/theme.xml
+    cp -v "$md_build/installer/themes/simple/art/icon.png" /etc/emulationstation/themes/simple/moonlight-embedded/art/icon.png
+    cp -v "$md_build/installer/themes/simple/art/logo.png" /etc/emulationstation/themes/simple/moonlight-embedded/art/logo.png
+    cp -v "$md_build/installer/themes/simple/art/background.png" /etc/emulationstation/themes/simple/moonlight-embedded/art/background.png

```

Comment:
Theme stuff should be maintained in the respective theme repositories, not installed in the script.